Sources/Formworks/FWForm/FWFormViewController.swift

```diff
@@ -0,0 +1,97 @@
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    //- MARK: Life cycle
+    public override func loadView() {
+        super.loadView()
+        view.backgroundColor = .fwFormBackground
+    }
+    
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpCollectionViewConstraints()
+    }
+    
+    /// This function will set up the layout of the CollectionView. It first configure
+    /// the item size that will be present on a group. And then configure
+    /// the group size so it specifies the portion of the screen that will occupy
+    @available(iOS 13.0, *)
+    private func setUpCollectionViewLayout() -> UICollectionViewCompositionalLayout {
+        
+        let itemLayoutSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(FormSpec.itemFractionalWidth),
+                                                    heightDimension: .fractionalHeight(FormSpec.itemFractionalHeight))
+        let item = NSCollectionLayoutItem(layoutSize: itemLayoutSize)
+        let groupLayoutSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(FormSpec.groupFractionalWidth),
+                                                     heightDimension: .fractionalHeight(FormSpec.groupFractionalHeight))
+        let group = NSCollectionLayoutGroup.horizontal(layoutSize: groupLayoutSize, subitems: [item])
+        group.contentInsets = NSDirectionalEdgeInsets(top: FormSpec.groupSpacingTop,
+                                                      leading: FormSpec.groupSpacingLeading,
+                                                      bottom: FormSpec.groupSpacingBottom,
+                                                      trailing: FormSpec.groupSpacingTrailing)
+        let section = NSCollectionLayoutSection(group: group)
+        return UICollectionViewCompositionalLayout(section: section)
+    }
+    
+    /// This function will create the necessary constraints for the CollectionView
+    /// to occupy the entire ViewController.
+    private func setUpCollectionViewConstraints() {
+        view.addSubview(formCollectionView)
+        let guides = view.safeAreaLayoutGuide
+        NSLayoutConstraint.activate([
+            formCollectionView.topAnchor.constraint(equalTo: guides.topAnchor),
+            formCollectionView.leftAnchor.constraint(equalTo: guides.leftAnchor),
+            formCollectionView.bottomAnchor.constraint(equalTo: guides.bottomAnchor),
+            formCollectionView.rightAnchor.constraint(equalTo: guides.rightAnchor)
+        ])
+    }
+}
+//- MARK: UICollectionViewDelegate
+extension FWFormViewController: UICollectionViewDelegate {
+    
+}
+//- MARL: UICollectionViewDataSource
+extension FWFormViewController: UICollectionViewDataSource {
+    
+    public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 30
```

Comment:
I understand this is for development purposes but don't forget to raise an issue about cell configuration later.

Reply:
Thanks for reminding this. I will create an issue stating that the cells need to be properly configurated.